code/datums/components/stationloving.dm

```diff
@@ -70,6 +70,8 @@
 	if (is_reserved_level(T.z))
 		if (is_type_in_typecache(A, allowed_shuttles))
 			return TRUE
+	if (is_away_level(T.z))
+		return FALSE
```

Comment:
Does this change anything? The next line returns FALSE anyway.

Reply:
Dayum, you are right. I tested it and nuke cannot be transported via gateway to away missions already. Odd, because last changes to this file are several months old before the issue was opened. Either way, closing this.